node/cmd/guardiand/node.go

```diff
@@ -812,8 +812,6 @@ func runNode(cmd *cobra.Command, args []string) {
 	logger.Info("Loaded guardian key", zap.String(
```

Comment:
<i class='graphite__hidden'>[Re: lines 211 to 211]<br/></i>

Since you added it, can you use p2p.DefaultPort here?<span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/2344?utm_source=unchanged-line-comment">Graphite</a>.</span>

Reply:
good idea, done